### PR TITLE
[ERSSUP-74034]-[]-[Added initial draft specification for create advic…

### PR DESCRIPTION
**Pull request by nhsd-jack-wainwright**
…e endpoint]-[JW]

## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

**Review by Nayira-Sanchez — COMMENTED** [review]
Review done with a couple small comments added.

File: sandbox/src/mocks/stu3/createAdvice/responses/ExampleAdvice.json [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+              {
+                "name": "clinicType",
+                "valueCoding": {
+                  "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-ClinicType-1",
+                  "code": "HEART_FAILURE"
+                }
+              },
+              {
+                "name": "ageAndGenderAppropriate",
+                "valueBoolean": true
```
[the remark this answers is not in the document]
Comment (nhsd-jack-wainwright) [reply sandbox/src/mocks/stu3/createAdvice/responses/ExampleAdvice.json:89]: Well spotted updated 👍

File: sandbox/src/mocks/stu3/createAdvice/responses/ExampleAdvice.json [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+              }
+            ],
+            "status": "current",
+            "mode": "snapshot",
+            "entry": [
+              {
+                "item": {
+                  "identifier": {
+                    "system": "http://fhir.nhs.net/Id/ers-service",
+                    "value": "11014"
```
[the remark this answers is not in the document]
Comment (nhsd-jack-wainwright) [reply sandbox/src/mocks/stu3/createAdvice/responses/ExampleAdvice.json:123]: Updated 🙂

**Review by Nayira-Sanchez — DISMISSED** [review]
Looks good 👍

**Review by Nayira-Sanchez — DISMISSED** [review]
Review done for the addition of endpoint description

**Pull request closed without merging**